web/src/api/config.ts

```diff
@@ -41,6 +41,7 @@ export const defaultConfig: Config = {
     captcha_type: 'none',
     password_min_entropy: 0,
     wizard_step: undefined,
+    themeSwitcher: 'visible',
```

Comment:
why don't you use boolean value? It's not obvious

Reply:
I guess I need to rename this variable to themeSwitcherState
By default it's 'visible'

Reply:
well I think we can keep it as themeSwitcher. It's okay if it has default value